account_wallet/wizards/account_payment_register.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright (C) 2022-Today: GRAP (http://www.grap.coop)
+    @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="account.view_account_payment_register_form" />
+        <field name="arch" type="xml">
+            <field name="journal_id" position="after">
+                <field name="account_wallet_id" />
```

Comment:
shouldn't you put some kind of domain ?

Reply:
Well, I was thinking that depending on accounting choice, it was possible to make many different accounts. 

you can add a domain, if you want. (but in the python part don't you think ?)